File: c#/7_kyu/to_jaden_case.cs
using System;
using System.Collections.Generic;

public static class JadenCase
{
  public static string ToJadenCase(this string phrase)
  {
    var words = phrase.Split(" ");
    var output = new List<string>();
    foreach (var word in words) 
      output.Add(char.ToUpper(word[0]) + word.Substring(1, word.Length - 1));
    return string.Join(" ", output);
  }
}

// https://www.codewars.com/kata/5390bac347d09b7da40006f6/train/csharp

File: javascript/7_kyu/to_jaden_case.js
// https://www.codewars.com/kata/5390bac347d09b7da40006f6/train/javascript

String.prototype.toJadenCase = function () {
  let output = [];
  for (let word of this.split(" "))
    output.push(`${word[0].toUpperCase()}${word.substring(1)}`);
  return output.join(" ");
};
